lcidlc/src/Parser.cpp

```diff
@@ -143,6 +143,8 @@ static bool ParserReport(ParserRef self, Position p_where, ParserError p_error,
 			fprintf(stderr, "%s'%s' expected\n", i == 0 ? "" : " or ", p_hints[i]);
 		}
 		break;
+    default:
```

Comment:
This should be `case kParserErrorNone:`.  You want to get a build warning (or better, a build error) when you add a new type of parser error but don't teach `ParserReport()` to handle it.